lib/server/src/statics.rs

```diff
@@ -0,0 +1,32 @@
+use lazy_static::lazy_static;
```

Comment:
Let's rename this file. I didn't mean for it to just be statics, but rather all the metrics stuff that can be pulled out of `server.rs`. I guess just `metrics.rs` unless you have a better suggestion?